go/pkg/cosmos/types.go

```diff
@@ -143,3 +142,52 @@ type Value struct {
 	// example: udenom
 	Denom string `json:"denom"`
 }
+
+type TxSearchResponse struct {
+	Jsonrpc string                 `json:"jsonrpc"`
+	Id      int32                  `json:"id"`
+	Result  TxSearchResponseResult `json:"result"`
+}
+
+type TxSearchResponseResult struct {
+	Txs        []TxSearchResponseResultTxs `json:"txs"`
+	TotalCount string                      `json:"total_count"`
+}
+
+type TxSearchResponseResultTxs struct {
+	Hash     *string                         `json:"hash,omitempty"`
+	Height   *string                         `json:"height,omitempty"`
+	Index    *int32                          `json:"index,omitempty"`
+	TxResult *TxSearchResponseResultTxResult `json:"tx_result,omitempty"`
+	Tx       *string                         `json:"tx,omitempty"`
+	Proof    *TxSearchResponseResultProof    `json:"proof,omitempty"`
+}
+
+// GetIndex returns the Index field value if set, zero value otherwise.
+func (o *TxSearchResponseResultTxs) GetIndex() int32 {
+	if o == nil || o.Index == nil {
+		var ret int32
+		return ret
+	}
+	return *o.Index
+}
```

Comment:
we aren't utilizing this so let's just remove
```suggestion
```

Reply:
this is used in coinstacks/cosmos/api/handlers.go and other places as well

Reply:
My bad, you are correct.